src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -1,8 +1,14 @@
 import { bindActionCreators } from 'redux';
 import { browserHistory } from 'react-router';
 import { connect } from 'react-redux';
-import { updateSearchQuery, searchWithAddress, searchWithBounds, fetchVAFacility } from '../actions';
-import { map, find, compact, isEmpty, debounce } from 'lodash';
+import {
+  updateSearchQuery,
+  genBBoxFromAddress,
+  searchWithBounds,
+  searchProviders,
+  fetchVAFacility
+} from '../actions';
+import { map, /* find, */ compact, isEmpty, debounce } from 'lodash';
```

Comment:
Yea, I left it in initially as I wasn't sure if my change was going to work out so I held on to it "just in case". It's gone in the latest version of the file.